imapclient/imapclient.py

```diff
@@ -28,6 +29,9 @@
     long = int  # long is just int in python3
 
 
+logger = getLogger(__name__)
+imaplib_logger = getLogger('imapclient.imaplib')
```

Comment:
Then the logger would be called `imapclient.imapclient.imaplib`. I initially named it only `imaplib` since it would be its name if imaplib was using logging.

What do you prefer?